src/modules/name/index.ts

```diff
@@ -182,19 +196,20 @@ export class Name {
     options: {
       firstName?: string;
       lastName?: string;
-      gender?: GenderType;
+      sex?: Sex | SexValue;
```

Comment:
Same here with gender property

+

```suggestion
      sex?: SexType;
```